modules/products/jira/locals.tf

```diff
@@ -43,12 +43,11 @@ locals {
     }
   }) : yamlencode({})
 
-  # After restoring the snapshot of the Jira database, a re-index is required. To avoid interruption in the Jira
-  # service we should exclude indexing status from the health check process.
-  # For more info see: https://jira.atlassian.com/browse/JRASERVER-66970
-  ignore_index_check = var.db_snapshot_id != null ? yamlencode({
+  # By default, Jira accepts an index snapshot taken within 24hours. In order to use snapshot older than 24hours we need to update following property value.
+  # It is set to 10 years.
+  reuse_old_index_snapshot = var.shared_home_snapshot_id != null ? yamlencode({
     jira = {
-      additionalJvmArgs = ["-Dcom.atlassian.jira.status.index.check=false"]
+      additionalJvmArgs = ["-Dcom.atlassian.jira.startup.max.age.of.usable.index.snapshot.in.hours=87600"]
```

Comment:
I dont even think Jira DC is ten years old 😂  Wouldnt a 1 year grace period be sufficient?

Reply:
It might be nice if this value was configurable. This means if there are snapshot expiry issues it can be resolved easily enough by extending the grace.

Reply:
I agree that 10 years is very generous figure. We can ask how often dc-apt team will release the share-home snapshot and calculate number from that. Key is to put enough hours to make sure jira picks up the snapshot

Reply:
It may be unnecessary to expose this value in config. If we do, vendors now have one more information to figure out. Isn't it easier if we just put enough buffer so the snapshot is always picked up?

Reply:
For Confluence I calculated the time from snapshot creation by adding another variable to define the snapshot creation date. I think this is the only way to be precise and don't make assumption.

Reply:
Having a long life cycle for index recovery files will prevent cleanup job to remove the older files, and if the environment is up and running for a some days we will have a new recovery index every day created by the scheduled job and eventually we run out of space in shared home. 

We should be wise to select this metric and as @errcode1202 suggested I think is good to make this configurable to be more precise in this area.

Reply:
I don't think we need to be precise. We just need to convince Jira that it should pick up any snapshot that is available in the shared home. We don't get any benefit from people configuring this property. The index either is present or not, when it is put together with the database and shared home from the same timestamp, it should be consistent.